cityofaustin/atd-airflow
# Maximo Work Orders Reporting DAG

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/16656

## Associated repo
https://github.com/cityofaustin/dts-maximo-reporting

## Testing

**Steps to test:**
1. be on VPN/city network
2. trigger this dag however you would like. May I suggest `$ docker compose run --rm airflow-cli dags test atd_row_reporting`
3. Check for success! Specifically from Socrata such as: 

> 2024-05-15 15:20:51,927 INFO: Getting work orders with start: 05/08/2024, end: 05/15/2024
> 2024-05-15 15:20:54,906 INFO: 1139 work orders found
> 2024-05-15 15:20:56,853 INFO: {'Errors': 0, 'Rows Created': 0, 'Rows Updated': 1139, 'Rows Deleted': 0}


---
#### Ship list
